static/js/modules/calendar-interval.js

```diff
@@ -0,0 +1,63 @@
+var moment = require('moment');
```

Comment:
I don't think we want to make modules anymore, this should be a service (yes, even though it doesn't have any IO, services are currently where all forms of "not a controller" live)